htdocs/Frameworks/moduleclasses/moduleadmin/moduleadmin.php

```
@@ -588,14 +588,44 @@ public function renderAbout($business = '', $logo_xoops = true)
               . "</td>\n"
               . "</tr>\n"
               . "</table>\n";
-        if ( true === $logo_xoops ) {
+        $ret .= $this->renderFooterInfo($logo_xoops);
```

PHPMD (2.15.0)

[error] The variable $logo_xoops is not named in camelCase. (undefined)

(CamelCaseVariableName)